src/main/java/model/TaschenrechnerModel.java

```diff
@@ -1,17 +1,24 @@
 package model;
 
 import exceptions.DivisionException;
+import extensible.functions.CalculatorFunction;
 import extensible.functions.CosFunction;
 import extensible.functions.SinFunction;
 import extensible.functions.TanFunction;
+import reflection.ExtensibleFunctionsLoader;
 
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
+import java.util.Set;
 
 public class TaschenrechnerModel {
```

Comment:
Könntest du die Klasse (und die Testklasse) in CalculatorModel umbenennen? Das müsste die merge-Konflikte auch beheben